src/controllers/dynakube/istio/serviceentry.go

```diff
@@ -71,11 +71,15 @@ func buildServiceEntryIP(meta metav1.ObjectMeta, host string, port uint32) *isti
 
 func handleIstioConfigurationForServiceEntry(istioConfig *configuration) (bool, error) {
 	probe, err := kubeobjects.KubernetesObjectProbe(ServiceEntryGVK, istioConfig.instance.GetNamespace(), istioConfig.name, istioConfig.reconciler.config)
-	if probe == kubeobjects.ProbeObjectFound {
+	switch probe {
+	case kubeobjects.ProbeObjectFound:
 		return false, nil
-	} else if probe == kubeobjects.ProbeUnknown {
+	case kubeobjects.ProbeUnknown:
 		log.Error(err, "istio: failed to query ServiceEntry")
 		return false, err
+	case kubeobjects.ProbeTypeNotFound:
```

Comment:
should this just be the `default` case?

Reply:
No, this should be correct, per default everything should be fine and nothing has to be done.

The probe type not found check is there to skip creation if istio CRDs were not installed.